components/layout/construct.rs

```diff
@@ -222,7 +248,7 @@ impl InlineFragmentsAccumulator {
                 pseudo: node.get_pseudo_element_type().strip(),
                 style: node.style(style_context),
                 selected_style: node.selected_style(),
-                flags: InlineFragmentNodeFlags::empty(),
+                flags: FIRST_FRAGMENT_OF_ELEMENT | LAST_FRAGMENT_OF_ELEMENT,
```

Comment:
Actually, I suppose information about positioning backgrounds won't be split at this stage. But, uh, you know... Something else could be. In theory.